src/qt/sendcoinsdialog.cpp

```diff
@@ -375,8 +358,39 @@ void SendCoinsDialog::send(QList<SendCoinsRecipient> recipients, QString strFee,
         formatted.append(recipientElement);
     }
 
+    // Limit number of displayed entries
+    int messageEntries = formatted.size();
+    int displayedEntries = 0;
+    for(int i = 0; i < formatted.size(); i++){
```

Comment:
This is a very strange way of limiting a list :P
Alternative:
```
int messageEntries = (int)formatted.size();
while (formatted.size() > MAX_SEND_POPUP_ENTRIES) {
    formatted.removeLast();
}
int displayedEntries = (int)formatted.size();
```

Reply:
Ahh now I see that this is just moved code. I'm fine with keeping the old code as this is not introduced in this PR.